setup.py

```diff
@@ -58,6 +58,11 @@ def read_file(filename, encoding='utf8'):
 else:
     install_requires.append('mock')
 
+if sys.version_info < (2, 7, 9):
```

Comment:
Here is the rationale about it: http://urllib3.readthedocs.org/en/latest/security.html#installing-urllib3-with-sni-support-and-certificates